core/trino-main/src/main/java/io/trino/server/security/PasswordAuthenticatorManager.java

```diff
@@ -29,19 +33,32 @@
 import static com.google.common.base.Preconditions.checkArgument;
 import static com.google.common.base.Preconditions.checkState;
 import static com.google.common.base.Strings.isNullOrEmpty;
+import static com.google.common.collect.ImmutableList.toImmutableList;
 import static io.airlift.configuration.ConfigurationLoader.loadPropertiesFrom;
+import static java.util.Arrays.stream;
 import static java.util.Objects.requireNonNull;
 
 public class PasswordAuthenticatorManager
 {
     private static final Logger log = Logger.get(PasswordAuthenticatorManager.class);
 
-    private static final File CONFIG_FILE = new File("etc/password-authenticator.properties");
+    private static final File DEFAULT_CONFIG_FILE = new File("etc/password-authenticator.properties");
```

Comment:
Can you move to as default value of `password-authenticator.config-files`?

Reply:
It appears to be screwing up testing pretty badly as the validation of the existence of the file is done always.